src/test/java/com/google/step/snippet/external/StackOverflowClientTest.java

```diff
@@ -100,7 +100,7 @@ public void noAnswerSearchTest() {
     Card actual =
         client.search(
             "https://stackoverflow.com/questions/44686609/implementing-a-neural-network-in-haskell",
-            "implement neural net in haskell");
+            "implement neural network in haskell");
```

Comment:
Was this an intended change?

Reply:
No, I think it was from a previous edit I made to the file. Removed.